src/flb_fstore.c

```diff
@@ -153,6 +269,9 @@ struct flb_fstore_file *flb_fstore_file_create(struct flb_fstore *fs,
         return NULL;
     }
 
+    fsf->_cache_head.next = NULL;
```

Comment:
this line and the next one are not required since the context creation is using `flb_calloc()` (let's try to avoid to manipulate linked list internals directly)

Reply:
Did not realize calloc used. Will avoid and remove.

Reply:
resolved.